build.sbt

```diff
@@ -30,7 +30,10 @@ val commonSettings2 = commonSettings ++ Seq(
 val commonSettings3 = commonSettings ++ Seq(
   scalaVersion := scala3,
   scalacOptions ++= Seq(
-    "-language:implicitConversions"
+    "-language:implicitConversions",
+    //"-rewrite", "-source", "3.4-migration",
+    "-Wconf:msg=.*will never be selected.*:silent"
```

Comment:
Do we really need this if the `@nowarn`s are added already below? It may be better to not have the generic warning suppression for later development

Reply:
there is one warning steeming from a "export" that the nowarn does not silence, unfortunately